test/api_test.go

```diff
@@ -0,0 +1,32 @@
+package test
+
+import (
+	"fmt"
+
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/cloud-native-toolkit/itzcli/api"
+	"github.com/stretchr/testify/assert"
+
+)
+
+func TestLogin(t *testing.T) {
+	r := api.SetUpRouter()
+	missingTokenreq, err := http.NewRequest("GET", "/login", nil)
+	if err != nil {
```

Comment:
You can save some work in the tests by using `assert.NoError(t, err)` here, and it will fail the test for you!